yapf/yapflib/file_resources.py

```diff
@@ -32,6 +32,35 @@
 CRLF = '\r\n'
 
 
+def _getExcludePatternsFromFile(filename):
+  ignore_patterns = []
+  # See if we have a .yapfignore file.
+  if os.path.isfile(filename) and os.access(filename, os.R_OK):
+    for line in open(filename, 'r').read().splitlines():
```

Comment:
Is it possible to use `open(filename, 'r').readlines()` instead?

Reply:
fixed